lib/perl/Genome/InstrumentData/Command/Import/WorkFlow/ResolveInstDataPropertiesFromCgHub.pm

```diff
@@ -0,0 +1,166 @@
+package Genome::InstrumentData::Command::Import::WorkFlow::ResolveInstDataPropertiesFromCgHub;
+
+use strict;
+use warnings;
+
+use Genome;
+
+use Genome::Model::Tools::CgHub::Query;
+require File::Spec;
+require File::Temp;
+use Path::Class;
+
+class Genome::InstrumentData::Command::Import::WorkFlow::ResolveInstDataPropertiesFromCgHub { 
+    is => 'Genome::InstrumentData::Command::Import::WorkFlow::ResolveInstDataProperties',
+};
+
+sub metadata_file {
```

Comment:
I think the names for `metadata_file` and `_resolve_metadata_file` could use clarification since both are doing logic to find a metadata file.